lib/services/monitorManagement/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "azure-arm-monitor",
   "author": "Microsoft Corporation",
   "description": "MonitorManagementClient Library with typescript type definitions for node",
-  "version": "4.2.0",
+  "version": "4.3.0",
```

Comment:
This should actually be a major version change because MetricCriteria inherited a required property from MultiMetricCriteria. This will break any current MetricCriteria objects that currently exist.